design/PROCESSES.md

```diff
@@ -0,0 +1,22 @@
+# Document chapters
+The following chapters/information must be included in the design document (according to the unofficial Snelting tips document):
```

Comment:
Will we talk somewhere about the design patterns that we use. This is also mentioned in the tips from Snelting and present in the example we got from Florian.

Reply:
Good catch.